xbmc/GUIInfoManager.h

```diff
@@ -95,7 +96,8 @@ class CSetCurrentItemJob;
  \brief
  */
 class CGUIInfoManager : public IMsgTargetCallback, public Observable,
-                        public KODI::MESSAGING::IMessageTarget
+                        public KODI::MESSAGING::IMessageTarget,
+                        public CJobQueue
```

Comment:
implementation details shold not be exposed to the world. better create a protected member for the jobQueue like I did for VP